common/app/common/TrailsToRss.scala

```diff
@@ -158,10 +158,6 @@ object TrailsToRss {
       pressedPage.collections
         .filterNot(_.config.excludeFromRss)
         .flatMap(_.curatedPlusBackfillDeduplicated)
-        .filter {
-          case _: LinkSnap => false
-          case _           => true
-        }
```

Comment:
Could we elaborate on why this has been dropped?

Reply:
For me it seems this has nothing to do with the Play upgrade?

Reply:
This is coming from the upgrade of the scala version, which through improvement of types pattern matching can detect that a value of type `model.pressed.PressedContent` can not also be a `com.gu.facia.api.models.LinkSnap`. The filtering code is therefore unnecessary and can be safely removed.

Reply:
We can't just keep that code, as it will be considered a compile error.